independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4553 @@
+                    },
+                    "hubspotscore": {
+                      "title": "HubSpot score",
+                      "type": "string",
+                      "description": "HubSpot score."
+                    },
+                    "industry": {
+                      "title": "Industry",
+                      "type": "string",
+                      "description": "Industry."
+                    },
+                    "ip_city": {
+                      "title": "IP city",
+                      "type": "string",
+                      "description": "IP city"
+                    },
+                    "ip_country": {
+                      "title": "IP country",
+                      "type": "string",
+                      "description": "IP country"
+                    },
+                    "ip_country_code": {
+                      "title": "IP country code",
+                      "type": "string",
+                      "description": "IP country code"
+                    },
+                    "ip_state": {
+                      "title": "IP state",
+                      "type": "string",
+                      "description": "IP state"
+                    },
+                    "ip_state_code": {
+                      "title": "IP state code",
+                      "type": "string",
+                      "description": "IP state code"
+                    },
+                    "job_function": {
+                      "title": "Job function",
+                      "type": "string",
+                      "description": "Job function."
+                    },
+                    "jobtitle": {
+                      "title": "Job title",
+                      "type": "string",
+                      "description": "Job title."
+                    },
+                    "lastmodifieddate": {
+                      "title": "Last modified date",
+                      "type": "string",
+                      "description": "Last modified date."
+                    },
+                    "lastname": {
+                      "title": "Last name",
+                      "type": "string",
+                      "description": "Last name."
+                    },
+                    "lifecyclestage": {
+                      "title": "Lifecycle stage",
+                      "type": "string",
+                      "description": "The current lifecycle stage."
+                    },
+                    "marital_status": {
+                      "title": "Marital status",
+                      "type": "string",
+                      "description": "Marital status."
+                    },
+                    "message": {
+                      "title": "Message",
+                      "type": "string",
+                      "description": "Message."
+                    },
+                    "military_status": {
+                      "title": "Military statusMilitary status",

```

Comment:
@troystaylor Typo? All the typos in swagger need to be fixed.

Reply:
Fixed